_apidocs/searchgov-results/v2/openapi.yml

```diff
@@ -70,6 +70,48 @@ paths:
           schema:
             type: string
             format: string
+        - name: audience
+          in: query
+          description: 'Allows you to filter results based on the Audience field. Accepts a comma-separated list of strings.'
+          required: false
+          schema:
+            type: string
+            format: string
+        - name: content_type
+          in: query
+          description: 'Allows you to filter results based on the Content Type field. Accepts a comma-separated list of strings.'
+          required: false
+          schema:
+            type: string
+            format: string
+        - name: tags
```

Comment:
`tags` is below as well, `mime_type` is missing though.

Reply:
guessing this should be `mime_type`?

Reply:
the tags below is different, it's not nested within the parameters object.
added mime_type